osu.Game.Rulesets.Catch/Objects/Drawables/DrawableBanana.cs

```diff
@@ -21,21 +20,32 @@ public DrawableBanana([CanBeNull] Banana h)
         {
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            RandomSeed.BindValueChanged(_ =>
+            {
+                UpdateComboColour();
+                UpdateInitialTransforms();
```

Comment:
Not sure how safe this is, since this method was only intended to be called by `DrawableHitObject` itself.

Is it feasible to just note make this see a bindable and assert that it is only changed on construction or pool retrieval instead?

Reply:
I'm not sure what your expected code. Is it better now?